IdentityCore/src/validation/MSIDAuthority.m

```diff
@@ -149,6 +150,16 @@ - (NSURL *)universalAuthorityURL
     return @[self.environment];
 }
 
+- (NSString *)enrollmentIdForHomeAccountId:(NSString *)homeAccountId
```

Comment:
Shouldn't this be present in AAD authority only?

Reply:
By the design doc and the ADAL code snippets, it looks like we only exclude ADFS authorities. If this behavior is not correct, I will be happy to fix :)

Reply:
ADAL only supports ADFS and AAD authorities, whereas MSAL also supports B2C :) Enrollment ID will be only applicable to AAD.

Reply:
I see, this makes sense then. Thanks!